base/all-all/sbhuddef.lmp

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
What exactly is the difference between `sbhuddef` and `sbardef` again?

Reply:
There are no HUD widgets in `sbardef` v1.0.0 (Legacy of Rust), so we will have to load them somehow. `sbhuddef` is only HUD widgets, previously it was a table in the code.

Reply:
Alright, thanks!